flink-runtime/src/test/java/org/apache/flink/runtime/io/network/netty/NettyMessageClientSideSerializationTest.java

```diff
@@ -183,15 +192,18 @@ private void testEncodeDecodeBuffer(boolean testReadOnlyBuffer, boolean testComp
 		}
 
 		NettyMessage.BufferResponse expected = new NettyMessage.BufferResponse(
-			testBuffer, random.nextInt(), new InputChannelID(), random.nextInt());
-		NettyMessage.BufferResponse actual = encodeAndDecode(expected);
+			testBuffer, random.nextInt(), inputChannel.getInputChannelId(), random.nextInt());
+		NettyMessage.BufferResponse actual = encodeAndDecode(expected, channel);
 
 		// Netty 4.1 is not copying the messages, but retaining slices of them. BufferResponse actual is in this case
```

Comment:
This comment is not invalid now? because we do not use previous `LengthFieldBasedFrameDecoder`

Reply:
Removed the comments.